msgid ""
msgstr ""
"Project-Id-Version: kolibri\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-06-11 09:07-0700\n"
"PO-Revision-Date: 2020-07-27 17:56\n"
"Last-Translator: \n"
"Language-Team: German\n"
"Language: de_DE\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: kolibri\n"
"X-Crowdin-Project-ID: 201933\n"
"X-Crowdin-Language: de\n"
"X-Crowdin-File: /release-v0.14.x/django.po\n"
"X-Crowdin-File-ID: 3806\n"

#: kolibri/core/auth/constants/collection_kinds.py:15
msgid "Facility"
msgstr "Facility"

#: kolibri/core/auth/constants/collection_kinds.py:16
msgid "Classroom"
msgstr "Klassenzimmer"

#: kolibri/core/auth/constants/collection_kinds.py:17
msgid "Learner group"
msgstr "Lerngruppe"

#: kolibri/core/auth/constants/collection_kinds.py:18
msgid "Ad hoc learners group"
msgstr "Ad-hoc-Lerngruppe"

#: kolibri/core/auth/constants/commands_errors.py:17
msgid "Unexpected exception [{}]: {}"
msgstr "Unerwarteter Ausnahmefehler [{}]: {}"

#: kolibri/core/auth/constants/commands_errors.py:20
msgctxt "Error report message when the allowed number of digits has been exceeded."
msgid "Content of cell '{}' is too long"
msgstr "Inhalt der Zelle '{}' ist zu lang"

#: kolibri/core/auth/constants/commands_errors.py:24
msgctxt "Error report message when the cell contains an improper data format."
msgid "Content of cell '{}' is not valid"
msgstr "Inhalt der Zelle '{}' ist ungültig"

#: kolibri/core/auth/constants/commands_errors.py:26
msgid "Duplicated username"
msgstr "Benutzername bereits vorhanden"

#: kolibri/core/auth/constants/commands_errors.py:28
#: kolibri/core/auth/management/commands/bulkimportusers.py:84
msgid "Username only can contain characters, numbers and underscores"
msgstr "Benutzername darf nur Buchstaben, Ziffern und Unterstriche enthalten"

#: kolibri/core/auth/constants/commands_errors.py:30
msgid "The column '{}' is required"
msgstr "Die Spalte '{}' ist erforderlich"

#: kolibri/core/auth/constants/commands_errors.py:31
msgid "Mix of valid and/or invalid header labels found in first row"
msgstr "Einige ungültige Spaltenüberschriften in der ersten Zeile gefunden"

#: kolibri/core/auth/constants/commands_errors.py:33
msgid "No default facility exists. Make sure to set up a facility on the device before running this command"
msgstr "Es ist keine standardmäßige Einrichtung vorhanden. Vor dem Ausführen dieses Befehls sollte eine Einrichtung auf dem Gerät erstellt werden."

#: kolibri/core/auth/constants/commands_errors.py:35
#: kolibri/core/auth/management/commands/bulkimportusers.py:98
msgid "Error trying to read csv file: {}"
msgstr "Fehler beim Lesen der CSV-Datei: {}"

#: kolibri/core/auth/constants/commands_errors.py:36
#: kolibri/core/auth/management/commands/bulkimportusers.py:99
msgid "Error trying to write csv file: {}"
msgstr "Fehler beim Schreiben der CSV-Datei: {}"

#: kolibri/core/auth/management/commands/bulkexportusers.py:43
#: kolibri/core/auth/management/commands/bulkexportusers.py:125
msgid "Database ID ({})"
msgstr "Datenbank-ID ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:44
#: kolibri/core/auth/management/commands/bulkexportusers.py:126
msgid "Username ({})"
msgstr "Benutzername ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:45
#: kolibri/core/auth/management/commands/bulkexportusers.py:127
msgid "Password ({})"
msgstr "Passwort ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:46
#: kolibri/core/auth/management/commands/bulkexportusers.py:128
msgid "Full name ({})"
msgstr "Vollständiger Name ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:51
msgctxt "CSV column header for the type of user: ADMIN, LEARNER, COACH..."
msgid "User type ({})"
msgstr "Benutzertyp ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:54
#: kolibri/core/auth/management/commands/bulkexportusers.py:136
msgid "Identifier ({})"
msgstr "Kennung ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:55
#: kolibri/core/auth/management/commands/bulkexportusers.py:137
msgid "Birth year ({})"
msgstr "Geburtsjahr ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:56
#: kolibri/core/auth/management/commands/bulkexportusers.py:138
msgid "Gender ({})"
msgstr "Geschlecht ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:61
msgctxt "CSV column header for the list of classrooms names where the learner is going to be enrolled"
msgid "Learner enrollment ({})"
msgstr "Einschreibung des Lernenden ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:68
msgctxt "CSV column header for the list of classrooms names where the tutor is going to be a coach"
msgid "Coach assignment ({})"
msgstr "Trainerzuweisung ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:133
msgctxt "CSV header for the type of user: ADMIN, LEARNER, COACH..."
msgid "User type ({})"
msgstr "Benutzertyp ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:143
msgctxt "CSV file header for the list of classrooms names where the learner is going to be enrolled"
msgid "Learner enrollment ({})"
msgstr "Einschreibung des Lernenden ({})"

#: kolibri/core/auth/management/commands/bulkexportusers.py:150
msgctxt "CSV file header for the list of classrooms names where the tutor is going to be a coach"
msgid "Coach assignment ({})"
msgstr "Trainerzuweisung ({})"

#: kolibri/core/auth/management/commands/bulkimportusers.py:75
msgctxt "Error message that might appear when there's a programming error importing a CSV file"
msgid "Unexpected error [{}]: {}"
msgstr "Unerwarteter Fehler [{}]: {}"

#: kolibri/core/auth/management/commands/bulkimportusers.py:79
msgctxt "Error when the command is executed in the Terminal (command prompt)"
msgid "Value in column '{}' has too many characters"
msgstr "Wert in Spalte '{}' hat zu viele Zeichen"

#: kolibri/core/auth/management/commands/bulkimportusers.py:81
msgid "Invalid value in column '{}'"
msgstr "Ungültiger Wert in Spalte '{}'"

#: kolibri/core/auth/management/commands/bulkimportusers.py:82
msgid "Username is duplicated"
msgstr "Der Benutzername ist bereits vorhanden"

#: kolibri/core/auth/management/commands/bulkimportusers.py:88
msgctxt "Error message indicating that the CSV file selected for import is missing a required column"
msgid "The column '{}' is required"
msgstr "Die Spalte '{}' ist erforderlich"

#: kolibri/core/auth/management/commands/bulkimportusers.py:92
msgctxt "Error message indicating that one column header in the CSV file selected for import is missing or incorrect"
msgid "Invalid header label found in the first row"
msgstr "Ungültige Spaltenüberschrift in der ersten Zeile gefunden"

#: kolibri/core/auth/management/commands/bulkimportusers.py:96
msgctxt "Error when the command is executed in the Terminal (command prompt)"
msgid "No default facility exists. Make sure to set up a facility on the  device before importing users and classes"
msgstr "Es ist keine standardmäßige Einrichtung vorhanden. Vor dem Importieren der Benutzer und Klassen sollte eine Einrichtung auf dem Gerät erstellt werden."

#: kolibri/core/auth/management/commands/bulkimportusers.py:101
msgid "The password field is required. To leave the password unchanged in existing users, insert an asterisk (*)"
msgstr "Die Eingabe des Passworts ist erforderlich. Um das Passwort vorhandener Benutzer beizubehalten, geben Sie ein Sternchen (*) ein."

#: kolibri/core/auth/management/commands/bulkimportusers.py:104
msgid "Cannot update user with ID: '{}' because no user with that database ID exists in this facility"
msgstr "Benutzer mit der ID '{}' kann nicht aktualisiert werden, da kein Benutzer mit dieser Datenbank-ID in dieser Einrichtung vorhanden ist."

#: kolibri/core/auth/management/commands/bulkimportusers.py:106
msgid "Database ID is not valid"
msgstr "Datenbank-ID ist ungültig"

#: kolibri/core/content/api.py:149
msgid "Resource"
msgstr "Material"

#: kolibri/core/content/api.py:910
msgid "The requested channel does not exist on the content server"
msgstr "Der angeforderte Kanal ist nicht auf dem Content-Server vorhanden"

#: kolibri/core/content/models.py:165
msgid "Unknown format"
msgstr "Unbekanntes Format"

#: kolibri/core/device/serializers.py:42
msgid "Language is not supported by Kolibri"
msgstr "Sprache wird von Kolibri nicht unterstützt"

#: kolibri/core/logger/csv_export.py:207
msgctxt "Default name for the exported CSV file with content session logs. Please keep the underscores between words in the translation"
msgid "content_session_logs"
msgstr "Inhalt_Sitzungsprotokolle"

#: kolibri/core/logger/csv_export.py:217
msgctxt "Default name for the exported CSV file with content summary logs. Please keep the underscores between words in the translation"
msgid "content_summary_logs"
msgstr "Inhalt_Übersichtsprotokolle"

#: kolibri/core/tasks/api.py:71
msgid "There was a network error."
msgstr "Ein Netzwerkfehler ist aufgetreten."

#: kolibri/core/tasks/api.py:73
msgid "There was a disk access error."
msgstr "Fehler beim Festplattenzugriff."

#: kolibri/core/tasks/api.py:75
msgid "There was an unknown error."
msgstr "Ein unbekannter Fehler ist aufgetreten."

#: kolibri/core/templates/kolibri/base.html:13
#: kolibri/core/templates/kolibri/unsupported_browser.html:13
msgid "Kolibri"
msgstr "Kolibri"

#: kolibri/core/templates/kolibri/unsupported_browser.html:42
msgid "Unsupported browser"
msgstr "Nicht unterstützter Browser"

#: kolibri/core/templates/kolibri/unsupported_browser.html:43
msgid "Sorry, your browser version is not supported."
msgstr "Ihre Browserversion wird leider nicht unterstützt."

#: kolibri/core/templates/kolibri/unsupported_browser.html:44
msgid "To use Kolibri, we recommend using Firefox or Chrome."
msgstr "Wir empfehlen Firefox oder Chrome für die Nutzung von Kolibri."

#: kolibri/core/templates/kolibri/unsupported_browser.html:45
msgid "You can also try updating your current browser."
msgstr "Möglicherweise können Sie den Fehler beheben, indem Sie Ihre Browserversion aktualisieren."

#: kolibri/core/views.py:157
msgid "No appropriate redirect pages found. It is likely that Kolibri is badly configured"
msgstr "Es wurden keine entsprechenden Weiterleitungsseiten gefunden. Möglicherweise wurde Kolibri nicht ordnungsgemäß konfiguriert."

#: kolibri/plugins/facility/views.py:46
msgctxt "Default name for the exported CSV file of facility user data. Please keep the underscore between words in the translation"
msgid "users_{}"
msgstr "Benutzer_{}"

#: kolibri/plugins/oidc_provider_plugin/templates/oidc_provider/authorize.html:7
msgid "OpenID Provider Authorization"
msgstr "Authorisierung des OpenID-Anbieters"

#: kolibri/plugins/oidc_provider_plugin/templates/oidc_provider/authorize.html:15
msgid "Request for Permission"
msgstr "Berechtigung anfordern"

